buddy-guide.md

```diff
@@ -0,0 +1,67 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Buddy Guide](#buddy-guide)
+- [Before the new student starts](#before-the-new-student-starts)
+- [Day 1](#day-1)
+- [Week 1](#week-1)
+- [End of Week 1](#end-of-week-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Buddy Guide
+
+As part of the onboarding process, each new student will be allocated a buddy. A buddy is typically an experienced team member who will be
+able to provide training, instruction and leadership for the new student in the crucial first number of weeks. As such, we need to provide
+a guide for what we expect of you as a buddy to help the initial days of the new students setup. This guide will not cover HOW to train
```

Comment:
I believe the buddy will be assigned to the student throughout the internship program? So this sentence here can be replaced with `...to help the student throughout their internship programme`